src/libs/actions/Policy.ts

```diff
@@ -3590,11 +3591,12 @@ function openPolicyDistanceRatesPage(policyID?: string) {
     API.read(READ_COMMANDS.OPEN_POLICY_DISTANCE_RATES_PAGE, params);
 }
 
-function navigateWhenEnableFeature(policyID: string, featureRoute: Route) {
+function navigateWhenEnableFeature(policyID: string, featureRouteName: string, featureRoute: Route) {
```

Comment:
Then we could use stricter type here too

Reply:
Not needed anymore since we ditched the route approach.